CONTRIBUTING.md

```diff
@@ -33,15 +34,15 @@ Thanks for wanting to contribute source code to ownCloud. That's great!
 Before we're able to merge your code to ownCloud Desktop Client, you need to sign
 our [Contributor Agreement][agreement].
 
-Please read the [Desktop Client Manual][desktopman] and the [Developer
-Manuals][devmanual] to get useful info like how to create your first
-application or how to test the ownCloud code with phpunit.
+Please read the [Desktop Client Manual][desktopman] to get useful info like
+how to create your first application or how to test the ownCloud code with
+phpunit.
```

Comment:
How to test the client with phpunit? What?

Reply:
I was wondering the same.... no idea what this is about ...

Reply:
... let me drop that part as well ...